.gitignore

```diff
@@ -19,3 +19,7 @@ cmake-build-debug
 *_REMOTE_*.txt
 *.orig
 
+setup.bash 
+setup.sh 
+setup.zsh
```

Comment:
What do these setup files normally contain? Might it be more appropriate to put them in the same location that we're installing the packages to (similar in structure to how `/opt/ros/kinetic` is structured)?

Reply:
Like mentioned, I haven't figured out how to merge the setup scripts cleanly yet.

Reply:
I will need to spend some time to actually look at the contents of the setup scripts or read some docs.